checks/agentVersionCheck_test.go

```diff
@@ -0,0 +1,40 @@
+package checks
+
+import (
+	"os"
+	"testing"
+
+	"github.com/newrelic/newrelic-lambda-extension/config"
+	"github.com/newrelic/newrelic-lambda-extension/lambda/extension/api"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAgentVersion(t *testing.T) {
+	conf := config.Configuration{}
+	reg := api.RegistrationResponse{}
+	r := runtimeConfig{}
+
+	// No version set
+	err := agentVersionCheck(&conf, &reg, r)
+	assert.Nil(t, err)
+
+	// Error
+	dirname, err := os.Getwd()
+	testFile := dirname + "/opt/python/lib/python3.8/site-packages/newrelic/"
+
+	r = runtimeConfigs[Python]
+	r.AgentVersion = "10.1.2"
+	r.layerAgentPaths = []string{testFile}
+	os.MkdirAll(testFile, os.ModePerm)
+	defer os.RemoveAll(dirname + "/opt")
```

Comment:
Can we make this path more specific? Something like `/opt/node/...`` or something like that?